tests/test_metric_cards.py

```
@@ -0,0 +1,59 @@
+        super().__init__()
+        self._state = state
+
+    def compose(self) -> ComposeResult:
+        yield MetricsRow(self._state)
+
+
+@pytest.mark.asyncio
+async def test_metric_card_displays_value():
+    """MetricCard should display label and value."""
+    app = MetricCardTestApp()
+    async with app.run_test() as pilot:
```

Ruff (0.14.13)

Local variable `pilot` is assigned to but never used

Remove assignment to unused variable `pilot`

(F841)